packages/@aws-cdk/aws-amplify/.npmignore

```diff
@@ -16,4 +16,7 @@ dist
 .LAST_PACKAGE
 .jsii
 
-*.tsbuildinfo
\ No newline at end of file
+*.tsbuildinfo
+
+# Include .jsii
+!.jsii
```

Comment:
Looks odd... we have .jsii excluded and then included? It should be included (!!)
Can we enforce with pkglint?